packages/nextjs/components/scaffold-eth/FaucetButton.tsx

```diff
@@ -21,7 +22,10 @@ const localWalletClient = createWalletClient({
  */
 export const FaucetButton = () => {
   const { address } = useAccount();
-  const { balance } = useAccountBalance(address);
+
+  const { data: balance } = useBalance({
+    address,
```

Comment:
I think we forget `watch:true` here

Reply:
I think that isn't required, as that is to watch an update to the wallet's balance right?

The purpose of obtaining the balance in FaucetButton is to check if balance is undefined or not, and as a result, assigns different classes to a div element. 

Could we even remove useBalance and use useAccount instead? Then basically checking on if the wallet is connected or not (Seems like that is what is going on, but useAccount might be a more pure solution).

I may be missing some context surrounding this so apologies if so.

Reply:
I was missing the context based on another comment of yours!

However, I still did not need to add watch: true for the tooltip to automatically go away. 

Regardless, I've added it as an extra safeguard: [8c7250ac79dcef157d737ede2d968c39c51eac7b](https://github.com/scaffold-eth/scaffold-eth-2/pull/788/commits/8c7250ac79dcef157d737ede2d968c39c51eac7b)